scripts/deploy.js

```diff
@@ -11,10 +11,10 @@ async function main() {
   const ONE_YEAR_IN_SECS = 365 * 24 * 60 * 60;
   const unlockTime = currentTimestampInSeconds + ONE_YEAR_IN_SECS;
 
-  const lockedAmount = hre.ethers.utils.parseEther("1");
-
   const SlotMachine = await hre.ethers.getContractFactory("SlotMachine");
-  const slotMachine = await SlotMachine.deploy();
+  const slotMachine = await SlotMachine.deploy({
+    value: hre.ethers.utils.parseEther("0.1"),
```

Comment:
what does this value map to in smart contract?

Reply:
This is essentially funding the contract with the required amount of ETH. In this case the value of the funding being 0.1.

Reply:
you mean the contract balance?

Reply:
yes. If you see in the previous iteration:
lockedAmount = hre.ethers.utils.parseEther("1");
was unused and the contract had no balance.

Reply:
Got it, I meant to ask this value=0.1 is the contract balance, correct?

Reply:
Yes